updating/updating-cluster-cli.adoc

```diff
@@ -8,6 +8,11 @@ toc::[]
 You can update, or upgrade, an {product-title} cluster by using the
 OpenShift CLI (`oc`).
 
+[IMPORTANT]
+====
+You can update your cluster to only a different minor version by using `oc`.
```

Comment:
You can tell your cluster to upgrade to whatever version using `oc adm upgrade ...`.  But telling it to take upgrades that aren't in our official Cincinnati graph (e.g. https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.1 ) may be unsupported.  I'm not clear on the official support policy in this space; maybe @crawford knows?

Reply:
It's unsupported. We really need to teach the CVO to do local validation as well so it's less easy to make that mistake.

Reply:
> We really need to teach the CVO to do local validation as well so it's less easy to make that mistake.

The CVO may already refuse non graph upgrades without `--force`.  But I was previously not clear on whether that override voided support for the cluster.  Even more reason to track edge removal in Git ;).

Reply:
I'll clarify that you can only update within the current minor version, but I'm not going to mention that they can make an unsupported update by using `--force`.

Reply:
> I'll clarify that you can only update within the current minor version...

Until we start recommending 4.1->4.2 upgrades.

> ... but I'm not going to mention that they can make an unsupported update by using `--force`.

But disconnected folks will have to do this, because the CVO cannot pull their signatures to validate them.

On the other hand, without `--force` it should be impossible to make an unsupported upgrade.

Reply:
> > I'll clarify that you can only update within the current minor version...
> 
> Until we start recommending 4.1->4.2 upgrades.

Clayton said that the best user experience was to upgrade 4.1 > 4.2 updates from the web console, so that's the only path I'm going to document.
> 
> > ... but I'm not going to mention that they can make an unsupported update by using `--force`.
> 
> But disconnected folks will have to do this, because the CVO cannot pull their signatures to validate them.
> 
> On the other hand, without `--force` it should be impossible to make an unsupported upgrade.

The disconnected update path will have its own separate topic after all my in-flight PRs merge.